lib/components/viewers/stop-viewer.js

```diff
@@ -285,6 +285,11 @@ class StopViewer extends Component {
           <FormattedMessage id="components.StopViewer.noStopsFound" />
         </div>
       )
+      // If geometries are available (and are not a point) and stop times 
+      // are not, it is a strong indication that the stop is a flex stop.
+      if (stopData?.geometries?.geoJson?.type !== 'Point') {
+        contents = (<div><FormattedMessage id="components.StopViewer.flexStop" /></div>)
```

Comment:
Remove surrounding parentheses.

Reply:
Removed in 6ffc02cb8262fbc542bc259d5275ddc5a62fb900